### PR TITLE
Add Romanian translation and fix English strings

### PR DESCRIPTION
**Pull request by tmihai20**
Added Romanian Translation and fixed English strings

### CODE REVIEW

File: resources/language/resource.language.en_gb/strings.po [hunk 1]
```diff
@@ -91,7 +91,7 @@ msgid "Open Youtube settings window"
 msgstr ""
 
 msgctxt "#30020"
-msgid "Execute setup"
+msgid "Execute setup wizard"
```
Comment (Twilight0) [inline resources/language/resource.language.en_gb/strings.po:94]: I would like this to remain as it was.

**Review by Twilight0 — CHANGES_REQUESTED** [review]

**Comment by Twilight0** [comment]
Thank you very much, I cherrypicked your commit regarding the romanian translation, so in a sense this PR has been merged.

**Comment by tmihai20** [comment]
Ok then, I will have to update my Romanian translation. You still have a typo inside the English strings, it had **el** instead of **en**.
Edit: I see you did that yourself. Be healthy and carry on :)

**Comment by Twilight0** [comment]
Yeah, I fixed it myself, thank you for your contribution once again.

**Pull request closed without merging**